package/src/monaco.contribution.ts

```diff
@@ -79,11 +79,11 @@ const defaultLanguageSettings: LanguageSettings = {
     syntaxErrorAsMarkDown: {
         enableSyntaxErrorAsMarkDown: false,
     },
-    enableQueryWarnings: false,
-    enableQuerySuggestions: false,
+    enableQueryWarnings: true,
```

Comment:
@noam-shapira Do you think we should change this?

Reply:
I don't mind leaving it. Maybe we should change after it's been in KWE and we get ok feedbacks.

Reply:
I agree a change to the defaults should be a big version change; but it's debatable because it's a user-facing change, not an api change.

Reply:
I'll revert it for now